src/seedu/addressbook/data/person/Address.java

```diff
@@ -9,13 +12,62 @@
 public class Address {
 
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in order of "
+                                                              + "BLOCK, STREET, UNIT, POSTAL_CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
-    public final String value;
     private boolean isPrivate;
+    
+    private final Block block;
+    private final Street street;
+    private final Unit unit;
+    private final PostalCode postalCode;
 
     /**
+     * Encapsulates one piece of address information
```

Comment:
A super class header comment should give more information than this. It should be clear to a developer who is extending your superclass at any point in time.